connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -342,6 +344,11 @@ message Expression {
   }
 }
 
+message ExpressionCommon {
```

Comment:
are we over-engineering? what else can be put here?

Reply:
It's same as `Relation.RelationCommon` so it's more for consistency (so we can reuse `Origin` as well for call site). I think it's fine.